core/src/main/java/com/sopt/core/designsystem/component/timepicker/NoostakTimePicker.kt

```diff
@@ -0,0 +1,333 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.sopt.core.R
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.core.designsystem.theme.NoostakTheme.colors
+import com.sopt.core.extension.noRippleClickable
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.map
+
+@Composable
+fun Picker2(
+    items: List<String>,
+    state: PickerState = rememberPickerState(),
+    modifier: Modifier = Modifier,
+    startIndex: Int = 0,
+    visibleItemsCount: Int = 3,
+    cornerShape: RoundedCornerShape = RoundedCornerShape(0.dp),
+    isLeft: Boolean = true
+) {
+    val visibleItemsMiddle = visibleItemsCount / 2
+    val listScrollCount = Integer.MAX_VALUE
+    val listScrollMiddle = listScrollCount / 2
+    val listStartIndex = listScrollMiddle - listScrollMiddle % items.size - visibleItemsMiddle + startIndex
+    fun getItem(index: Int) = items[index % items.size]
+    val listState = rememberLazyListState(initialFirstVisibleItemIndex = listStartIndex)
+    val flingBehavior = rememberSnapFlingBehavior(lazyListState = listState)
+
+    LaunchedEffect(listState) {
+        snapshotFlow { listState.firstVisibleItemIndex }
+            .map { index -> getItem(index + visibleItemsMiddle) }
+            .distinctUntilChanged()
+            .collect { item -> state.selectedItem = item }
+    }
+
+    Box(
+        modifier = modifier
+            .height(152.dp)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Spacer(modifier = Modifier.height(32.dp))
+            Box(
+                modifier = Modifier
+                    .padding(
+                        start = if (isLeft) 10.5.dp else 0.dp,
+                        end = if (!isLeft) 10.5.dp else 0.dp
+                    )
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(NoostakTheme.colors.gray200)
+            )
+            Spacer(modifier = Modifier.height(19.dp))
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(48.dp)
+                    .background(
+                        color = NoostakTheme.colors.blue100,
+                        shape = cornerShape
+                    )
+                    .padding(horizontal = 20.dp, vertical = 7.dp)
+            )
+            Spacer(modifier = Modifier.height(18.dp))
+            Box(
+                modifier = Modifier
+                    .padding(
+                        start = if (isLeft) 10.5.dp else 0.dp,
+                        end = if (!isLeft) 10.5.dp else 0.dp
+                    )
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(NoostakTheme.colors.gray200)
+            )
```

Comment:
P1: HorizontalDivider로 수정해주세요
```suggestion
            HorizontalDivider(
                modifier = Modifier
                    .fillMaxWidth()
                    .padding(
                        start = if (isLeft) 10.5.dp else 0.dp,
                        end = if (!isLeft) 10.5.dp else 0.dp
                    ),
                thickness = 1.dp,
                color = colors.gray200
            )
```

Reply:
수정띠예